docs/proposals/THV-2106-virtual-mcp-server.md

```diff
@@ -440,7 +440,12 @@ composite_tools:
   - name: "deploy_and_notify"
     description: "Deploy PR with user confirmation and notification"
     parameters:
-      pr_number: {type: "integer"}
+      type: object
```

Comment:
hah, this is probably tripping CI..good on you to change the proposal retroactively. We might need to override CI this time